GOVERNANCE.md

```diff
@@ -2,57 +2,126 @@ Governance of the Chapel Project
+  feature requests, bug reports, and feedback.
+
+Generally speaking, all committers are contributors, and any
+contributor or community collaborator can request to join the TSC.  At
+present, we are lacking a means for a community members to be a
+contributor, but one of the roles of the TSC will be to define this
+process.
+
+
+Technical Steering Committee (TSC)
+----------------------------------
+
+Role: The project's overall technical direction is governed by the
+Technical Steering Committee, whose members' roles are detailed in
+`CHARTER.md`.  At the inception of the project, TSC membership was
+offered to everyone with commit privileges to the chapel-lang/chapel
+repo.  This led to the current TSC membership, which is:
+
+* Jade Abraham
+* Brad Chamberlain
+* Lydia Duncan
+* Daniel Fedorin
+* Abhishek Girish
+* Ben Harshbarger
+* John Hartman
+* Engin Kayraklioglu
+* Shreyas Khandekar
+* Anna Rift
+* Elliot Ronaghan
+* Andy Stone
+* Karlon West
+
+See `CHARTER.md` for additional information about TSC membership,
+roles, and responsibilities.
+
+
+TSC Chair
+---------
+
+The TSC may decide to define a chair for the committee and elect
+someone to that role, with responsibilities such as presiding over
+meetings of the TSC and serving as the primary communication point
+between the Chapel project and the High Performance Software
+Foundation (HPSF).  The TSC Chair would serve until their resignation
+or replacement by the TSC.
+
+At the time of this writing, the TSC has not yet met and opted to do,
+so the position of chair is currently undefined and unfilled.
+
+
+Tech Lead
+---------
+
+Role: The project technical lead's role is to help guide decisions for
+the project in ways that are technically sound, responsive to user
+needs and concerns, cognizant of the project's history and past
+decisions, and compatible with other efforts being undertaken across
+the project.  With the formation of the TSC, the technical lead role
+may be supplanted or amended by the role of a TSC chair, but this has
+yet to be determined.
+
+[Brad Chamberlain](https://github.com/bradcray) currently holds this
+role, as a founding member of the Chapel project who has been
+continually involved with it since the outset.  Brad has served as the
+technical lead of the project since 2006, after inheriting the role
+from David Callahan.
+
+
+Subteam Leads
+-------------
 
 Role: As the scope of the Chapel project has grown, a number of
 subteams have been spun up to focus on specific initiatives at finer
 granularities.  Generally speaking, these subteam leads have a great
 deal of latitude in making decisions to meet their subteams' goals,
-but are expected to keep the technical lead aware of major decisions,
-particularly for those that are contentious, represent a significant
-change in direction, or have the possibility of impacting other
-subteams.
+but are expected to keep the TSC, through its chair, aware of major
```

Comment:
This assumes a chair. maybe "through the Tech Lead"? Even if we decide to have a TSC Chair role, Tech Lead seems to be the better role for this type of liaison to me.